core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -114,26 +112,45 @@ public DDF transformUDF(String RExp) throws DDFException {
    * @return "(arrtime - crsarrtime) as foobar, (distance / airtime) as speed
    */
 
-  public static String RToSqlUdf(String RExp, List<String> existingColumns) {
+  public static String RToSqlUdf(String RExp, List<String> selectedColumns, List<Column> existingColumns) {
```

Comment:
@nhanitvn We do support nativeR, but the user will have to specify "R". If "R" is specified, the expression is directly passed to transformNativeRserve. Again, I think here exist a mismatch between the API and what can be passed to the API. In this case, even though we specify "Hive" the expression we would give it is R, which doesn't really make sense to me.